### PR TITLE
change event model

### PR DESCRIPTION
**Pull request by huynhngocminh2511**

### CODE REVIEW

File: ClubManagement/ClubManagement/Models/UserLoginEventModel.cs [hunk 1]
```diff
@@ -11,16 +11,16 @@ public UserLoginEventModel()
...
-            base.CreatedBy = eventModel.CreatedBy;
-            base.CreatedTime = eventModel.CreatedTime;
-            base.Latitude = eventModel.Latitude;
-            base.Longitude = eventModel.Longitude;
-            base.Place = eventModel.Place;
-            base.ImageUrl = eventModel.ImageUrl;
+            Id = eventModel.Id;
+            Title = eventModel.Title;
+            Description = eventModel.Description;
+            TimeStart = eventModel.TimeStart;
```
Comment (voquanghoa) [inline ClubManagement/ClubManagement/Models/UserLoginEventModel.cs:17]: Please add `TimeEnd` to here and other places

Comment (huynhngocminh2511) [reply ClubManagement/ClubManagement/Models/UserLoginEventModel.cs:17]: Done

File: ClubManagement/ClubManagement/Fragments/EventDialogFragment.cs [hunk 2]
```diff
@@ -73,7 +73,7 @@ private void OnClickSave(object sender, EventArgs e)
                     Longitude = place.LatLng.Longitude,
                     Latitude = place.LatLng.Latitude,
                     Place = place.AddressFormatted.ToString(),
-                    Time = date,
+                    TimeStart = date,
```
Comment (voquanghoa) [inline ClubManagement/ClubManagement/Fragments/EventDialogFragment.cs]: Is `TimeEnd` missing here?

Comment (nguyenvanman) [reply ClubManagement/ClubManagement/Fragments/EventDialogFragment.cs]: this file is unused and will be removed in next version

Comment (huynhngocminh2511) [reply ClubManagement/ClubManagement/Fragments/EventDialogFragment.cs]: Done. This file was deleted

**Pull request merged**